pkg/ingester/ingester.go

```diff
@@ -312,6 +312,8 @@ type Ingester struct {
 
 	activeGroups *util.ActiveGroupsCleanupService
 
+	costAttribution *util.CostAttributionCleanupService
```

Comment:
I'd keep the name consistent:
```suggestion
	costAttributionSvc *util.CostAttributionCleanupService
```